src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -188,10 +199,18 @@ public void copyFile(File file) {
             ZipUtil.pack(file, tmp);
             byte[] archive = IOUtils.toByteArray(new FileInputStream(tmp));
 
-            addShellStep(String.format(
-                    "base64 --decode << ENDOFFILE > archive.zip && unzip -o archive.zip \n%s\nENDOFFILE",
-                    new String(Base64.encodeBase64Chunked(archive))
-            ));
+            if (SystemUtils.IS_OS_WINDOWS) {
+                if (controller instanceof RemoteJenkinsController) {
```

Comment:
Rather you want to check that it is _not_ `instanceof LocalController`: there are several kinds of controller that would not work with this trick.